packages/typescript-express-runtime/src/server.ts

```diff
@@ -81,7 +81,10 @@ export type ServerConfig = {
    * if disabling, ensure you pass a body parsing middleware that places the parsed
    * body on `req.body` for request body processing to work.
    **/
-  body?: "disabled" | OptionsJson | undefined
+  body?:
+    | "disabled"
+    | Partial<{json: OptionsJson; text: OptionsText}>
```

Comment:
This is the breaking change: `body: OptionsJson` -> `body: Partial<{json: OptionsJson}>`